relayer/tests/browser_test.ts

```diff
@@ -61,20 +62,20 @@ describe("Browser tests", () => {
   var poseidon: any;
   var provider: Provider;
   var user: User;
-  const walletMock = useWallet(ADMIN_AUTH_KEYPAIR);
-  const connection = new Connection("http://127.0.0.1:8899", "confirmed");
+  const walletMock = useWallet(ADMIN_AUTH_KEYPAIR, RPC_URL);
+  const connection = new Connection(RPC_URL, "confirmed");
 
   before(async () => {
     poseidon = await circomlibjs.buildPoseidonOpt();
 
     await createTestAccounts(connection);
 
     const relayerRecipientSol = SolanaKeypair.generate().publicKey;
-    await connection.requestAirdrop(relayerRecipientSol, 2_000_000_000);
+    await connection.requestAirdrop(relayerRecipientSol, 2 * AIRDROP_DECIMALS);
```

Comment:
happy for a SOL_DECIMALS constant in zk.js but I don't think it's good to start putting such constants in many different places.